tern/__main__.py

```diff
@@ -153,12 +168,18 @@ def main():
     parser_report.add_argument('-d', '--dockerfile', type=check_file_existence,
                                help="Dockerfile used to build the Docker"
                                " image")
-    parser_report.add_argument('-i', '--docker-image',
-                               help="Docker image that exists locally -"
+    parser_report.add_argument('-i', '--image',
+                               help="Image that exists locally -"
+                               "either can be a docker image with format"
                                " image:tag"
+                               " or an OCI image with format"
+                               " oci://<image-location>:<image-tag>"
                                " The option can be used to pull docker"
                                " images by digest as well -"
                                " <repo>@<digest-type>:<digest>")
+    parser_report.add_argument('-t', '--type',
```

Comment:
I would like to propose that we call this option`client` instead of `type`. This makes more sense to me as we would like to support the podman client in the future which represents image blogs differently than docker or skopeo might, even if they all are stored on dockerhub technically as OCI images.

Reply:
Agree - this makes better sense than the generic "type" in the container image context.